viewing_party/party.py

```diff
@@ -1,23 +1,131 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    new_movie ={}
+    if not title or not genre or not rating:
+        return None
+    else:    
+        new_movie["title"] = title 
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    return new_movie

```

Comment:
Let's move the `new_movie` creation into the else statement. That way we aren't just using up memory space in case it hits the if statement and immediately returns

```suggestion
    if not title or not genre or not rating:
        return None
    new_movie ={}
    new_movie["title"] = title 
    new_movie["genre"] = genre
    new_movie["rating"] = rating
    return new_movie
```